module/workflow/src/Infrastructure/Grid/StatusGridBuilder.php

```diff
@@ -75,7 +75,7 @@ private function getCodes(Language $language): array
     {
         $result = [];
         foreach ($this->statusQuery->getAllStatuses($language) as $code => $status) {
-            $result[] = new StatusOption($code, $code, new Color($status['color']), $status['name']);
```

Comment:
its be good also change  $code na $id, actual name is misleading  and that was probably actual error generator